### PR TITLE
Add Task2

### PR DESCRIPTION
**Pull request by KondratenkoAlexey**

### CODE REVIEW

File: Tasks/Kondratenko/Task2/Task2.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+             13: ["", "", "", "тринадцать минут", "тринадцати минут"],
+             14: ["", "", "", "четырнадцать минут", "четырнадцати минут"],
+             15: ["", "", "", "пятнадцать минут", "пятнадцати минут"],
+             16: ["", "", "", "шестнадцать минут"],
+             17: ["", "", "", "семнадцать минут"],
+             18: ["", "", "", "восемнадцать минут"],
+             19: ["", "", "", "девятнадцать минут"],
+             20: ["двадцать", "", "", "двадцать минут"],
+             30: ["тридцать", "", "", "тридцать минут"],
+             40: ["сорок", "", "", "сорок минут"]}
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kondratenko/Task2/Task2.py:25]: the data structure is very inventive , I like it

File: Tasks/Kondratenko/Task2/Task2.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+def time_conversion(time):
+    hour, minute = time.split(":")
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kondratenko/Task2/Task2.py:29]: some additional validation would be nice, what if user forget to input the ':' symbol? -1 point for the unwise choice

**Review by MikitaTsiarentsyeu — APPROVED** [review]
the solution is very good, you receive 100 points and 9 points go to the bank for the functions usage

File: Tasks/Kondratenko/Task2/Task2.py (hunk 2 above)
Comment (KondratenkoAlexey) [reply Tasks/Kondratenko/Task2/Task2.py:29]: > some additional validation would be nice, what if user forget to input the ':' symbol? -1 point for the unwise choice

@MikitaTsiarentsyeu  In my solution, before calling the function, there are several checks (the code is given below), including the presence of ":" between XX and MM. Therefore, in the body of the function itself, I am sure that the value will be entered in the format "HH:MM".
 
**while True:  # checking the entered value for conformity to the format
        input_time = input("Enter the time in the format HH:MM: ")
        if len(input_time) > 5:
            print("Repeat input (too long value)")
        elif len(input_time) < 5:
            print("Repeat input (too short value)")
        _elif input_time[2] != ":":
            print("Repeat input (must be a character ':')")_
        elif not input_time.split(":")[0].isdigit() or not 0 <= int(input_time.split(":")[0]) < 24:
            print("Repeat input ('HH' must consist of numbers from 0 to 23)")
        elif not input_time.split(":")[1].isdigit() or not 0 <= int(input_time.split(":")[1]) < 60:
            print("Repeat input ('MM' must consist of numbers from 0 to 59)")
        else:
            break**

Comment (MikitaTsiarentsyeu) [reply Tasks/Kondratenko/Task2/Task2.py:29]: ok, missed it, 1 point returns to you

**Pull request merged**